ui/js/models/perf/series.js

```diff
@@ -87,6 +87,17 @@ treeherder.factory('PhSeries', ['$http', 'thServiceDomain', 'ThOptionCollectionM
                                  }
                                  return $q.reject("No series data found");
                              });
+        },
+        getReplicateData: function (params) {
+            params.value = 'perfherder-data.json';
+            return $http.get(thServiceDomain + '/api/jobdetail/'
+                , {params: params }).then(
```

Comment:
Some spacing issues here (probably need to update the eslint rules to catch these automatically...)